portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -12,21 +12,61 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// TODO: Modify servlet to access multiple messages from servlet.
+
 package com.google.sps.servlets;
 
+import java.util.*; 
+import com.google.gson.Gson;
 import java.io.IOException;
+import com.google.sps.data.Comment;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns "Hello Jay Saleh!". TODO: modify this file to handle comments data */
+/** Servlet that writes an list of messages as a response. */
+// TODO: Modify servlet to store values in database.
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  
+  /** Hard coded messages used for tesing JSON */
+  private List<Comment> messages;
+
+  @Override
+  public void init() {
+    messages = new ArrayList<>();
```

Comment:
ImmutableList.of()

Please add 
```
<dependency>
  <groupId>com.google.guava</groupId>
  <artifactId>guava</artifactId>
  <version>29.0-jre</version>
</dependency>
```

To your pom.xml

Reply:
I don't think I can use ImmutableList here because I need to be able to add comments to `messages`